### ISSUE TITLE
`<input type="month">` - `IDL attributes` : missing `valueAsDate` and `valueAsNumer`

### ISSUE THREAD

**Issue by Malix-off** [mdn/browser-compat-data#21873]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/month

### What specific section or headline is this issue about?

[Technical Summary](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/month#technical_summary)

### What information was incorrect, unhelpful, or incomplete?

About `<input type="month">` : `IDL attributes` - missing `valueAsDate` and `valueAsNumer`

### What did you expect to see?

About `<input type="month">` : `IDL attributes` - missing `valueAsDate` and `valueAsNumer`

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/input/month`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/month
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/input/month/index.md
* Last commit: https://github.com/mdn/content/commit/e04d8d2766c468f149445c0bf438d09f9b2d188c
* Document last modified: 2023-06-13T08:10:47.000Z

</details>

**Comment by hamishwillee** [comment]
BCD team - the two HTML input values `valueAsDate` and `valueAsNumber` [are not in BCD](htps://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#browser_compatibility). As per above, they may be supported in some browsers. If so, we can document them once BCD confirms this and is updated.

**Comment by queengooborg** [comment]
Actually, this is a case of looking at the wrong page/table.  The IDL values are tracked on the [API page](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement), not the HTML element page.  If you look at the HTMLInputElement API page, you'll see both `valueAsDate` and `valueAsNumber` there.

**Comment by Malix-off** [comment]
@queengooborg yes, but not there: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/month#technical_summary.

**Comment by queengooborg** [comment]
You're looking on the wrong page.  You want to look at https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement#browser_compatibility instead, which describes the interface and its features.  We don't document IDL attributes in the HTML data; we only document HTML attributes there.

**Comment by Malix-off** [comment]
@queengooborg 

> You're looking on the wrong page. You want to look at [developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement#browser_compatibility](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement#browser_compatibility) instead, which describes the interface and its features. We don't document IDL attributes in the HTML data; we only document HTML attributes there.

Why are there in every other input elements, then?

**Comment by queengooborg** [comment]
Oh, wait- you're not talking about the browser compatibility section.
